TweetMate/src/main/java/com/ybsystem/tweetmate/fragments/dialog/PreviewAppearanceDialog.java

```diff
@@ -36,17 +33,10 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
         // Inflate
         View view = inflater.inflate(R.layout.dialog_preview_appearance, null);
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ybs32/TweetMate-for-Android/blob/61661c239c012b88a18da6118962b9568a3a96d6/TweetMate/src/main/java/com/ybsystem/tweetmate/fragments/dialog/PreviewAppearanceDialog.java#L34"></span>Avoid passing <code>null</code> as the view root (needed to resolve layout parameters on the inflated layout's root element)</td>
    </tr>
  </tbody>
</table>